XCUITests/BookmarkingTests.swift

```diff
@@ -97,6 +97,36 @@ class BookmarkingTests: BaseTestCase {
         navigator.goto(MobileBookmarks)
         checkItemInBookmarkList()
     }
+    
+    func testRecentBookmarks() {
+        // Verify that there are only 4 cells without recent bookmarks
+        navigator.goto(LibraryPanel_Bookmarks)
+        XCTAssertEqual(app.tables["Bookmarks List"].cells.count, 4)
+        
+        //Add a bookmark
+        navigator.goto(URLBarOpen)
```

Comment:
Is this necessary? I was wondering if just with the next line `navigator.openURL("https://wikipedia.org")` it would work.
I would also add the url as a variable instead of directly adding it to the openURL